.goreleaser.ubuntu-latest.yml

```diff
@@ -0,0 +1,36 @@
+---
+project_name: saml2aws
+
+builds:
+- main: ./cmd/saml2aws/main.go
+  binary: saml2aws
+  flags:
+    - -trimpath
+    - -v
+  ldflags:
+    - -s -w -X main.Version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}
+  goos:
+    - windows
+    - linux
+  goarch:
+    - amd64
+    - arm64
+    - arm
+  overrides:
+    - goos: linux
+      goarch: amd64
+      goamd64: v1
+      env:
+        - CGO_ENABLED=0
```

Comment:
would building both CGO_ENABLED=0 and CGO_ENABLED=1 on `ubuntu-20.04` help compatibility (with ubuntu-20.04+)?

Reply:
No, that won't make any difference as the resulting binary is static and therefore has no dependencies on OS libraries.

Also, trying to build both on 20.04 would make the actions workflow more complicated as you wouldn't be able to just use matrix.os everywhere (you would have two different goreleaser config files for the same build OS). Building U2F on 20.04 doesn't just improve compatibility, it also make the workflows nicer :)

Reply:
The two Linux builds (goreleaser configs) also have different project names to produce differently named artefacts.